testing/scenario_app/run_android_tests.sh

```diff
@@ -29,7 +29,12 @@ function follow_links() (
 )
 
 SCRIPT_DIR=$(follow_links "$(dirname -- "${BASH_SOURCE[0]}")")
+SRC_DIR="$(cd "$SCRIPT_DIR/../../.."; pwd -P)"
+OUT_DIR="$SRC_DIR/out/$1"
 
 cd "$SCRIPT_DIR/android"
 GRADLE_USER_HOME="$PWD/android/gradle-home/.cache"
-set -o pipefail && ./gradlew app:verifyDebugAndroidTestScreenshotTest --gradle-user-home "$GRADLE_USER_HOME"
+set -o pipefail && ./gradlew app:verifyDebugAndroidTestScreenshotTest \
```

Comment:
it can be moved to gn too, right?

Reply:
This actually runs unit tests. GN builds things, it doesn't run them.